tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/services/ServiceDeclTest.java

```diff
@@ -114,4 +114,13 @@ public void testServiceDeclAndListenerAttachmentsNegative() {
                 "(ListenerWithNonNilInitReturnType|Int)'", 264, 12);
         Assert.assertEquals(i, result.getErrorCount());
     }
+
+
+    @Test
+    public void testRelativePathStartsWithSlashSyntaxError() {
+        CompileResult compileResult = BCompileUtil.compile("test-src/services/" +
+                "service_relative_resource_path_negative.bal");
+        Assert.assertEquals(compileResult.getErrorCount(), 1);
+        validateError(compileResult, 0, "relative resource path should not begin with slash", 35, 27);
```

Comment:
@MaryamZi @hasithaa any thoughts on the error msg. Looks good?

Reply:
Can't we just say `resource path should not begin with slash`?

I also prefer "cannot" over "should not".

```
resource path cannot begin with slash
```

Reply:
In service-decl we have absolute resource path which has to be started with '/'. So if we did not say explicitly "relative resource path"can someone mistake this error msg for resource path in both places cannot begin with a slash?

Reply:
The word `relative` is not correct here. Just saying `resource path` is enough. In IDEs, we highlighted the right range, so they know the actual error.

Reply:
Updated